admin/client/src/styles/_variables.scss

```diff
@@ -697,3 +699,36 @@ $cms-panel-xs: 60px;
 $cms-panel-sm: 300px;
 $cms-panel-md: 448px;
 $cms-panel-lg: 638px;
+
+// Shadows, these need work to make consistent (used in AssetAdmin)
+$shadow-light: rgba(0, 0, 0, .2);
+$dropdown-shadow: 0 0 3px $shadow-light;
+$gallery-item-shadow: 0 1px 4px $gray-light;
+
+// Masks: typically behind modals, drag drop areas
+$page-mask-light: rgba(236, 239, 241, .9);
+
+
+// Variables for use within but not limited to the AssetAdmin module
```

Comment:
They are currently only used in AssetAdmin but they should be generic enough for use elsewhere. The note is there just incase they don't get used again in framework and someone want to delete them—its a reminder to keep them even though they might look like they aren't used.

I can remove comment if you think its not needed?